apps/aecore/src/aec_governance.erl

```diff
@@ -233,21 +233,14 @@ add_network_id_last(Payload) ->
     NetworkId = get_network_id(),
     <<Payload/binary, NetworkId/binary>>.
 
--ifdef(TEST).
 get_network_id() ->
-    %% Needed for eunit
+    %% Easy switch networks with console argument, it overwrites config
     case init:get_argument(network_id) of
```

Comment:
Needed? 🤔 
```
$ docker run --rm -it --entrypoint bin/aeternity --network=none aeternity/aeternity:v3.1.0 console -aecore network_id '<<"foo">>'
...
(aeternity@localhost)1> aec_governance:get_network_id().
<<"foo">>
```

Reply:
I find `'<<"foo">>'`syntax  brain damaging, so I reused what was planted by @gorillainduction in the tests :)

Reply:
I see no need to plant yet another vector for configuration.

Reply:
If you mean the `'<<"foo">>'` cryptics I don't agree.

I tried to make it work as `-aecore network_id ae_uat` without luck as it's always read as atom.

Reply:
I think this is reasonable for future proofing the eunit tests. The eunit framework does not easily take hints, and the init arg was what I got working in the past.